Resources/translations/messages.nl.yml

```diff
@@ -0,0 +1,32 @@
+Properties: Eigenschappen
+# tabs and accordions
+pane.accordion:
+    general_properties: Algemene Eigenschappen
+
+pim_custom_entity:
+    export:
+        limit_exceeded: Selecteer alstublieft minder dan %limit% akten
```

Comment:
What sentence did you try to translate here? I am mostly curious to which word was the source of 'akten'. 
I am curious if it can be 'replaced' with 'regels' or something more common than 'akten'